files/RPM-GPG-KEY-emr

```diff
@@ -0,0 +1,30 @@
+-----BEGIN PGP PUBLIC KEY BLOCK-----
```

Comment:
What is this thing?

Reply:
I just always get a bit jumpy when I see keys in a repo and I don't remember seeing this in our original CR.